test/parallel/test-typescript.js

```diff
@@ -52,12 +52,9 @@ test('execute a typescript file with imports with no extensions', async () => {
     fixtures.path('typescript/test-import-no-extension.ts'),
   ]);
 
-  deepStrictEqual(result, {
-    code: 0,
-    signal: null,
-    stderr: '',
-    stdout: 'Hello, TypeScript!\n',
-  });
+  ok(result.stderr.includes('Error [ERR_MODULE_NOT_FOUND]:'));
```

Comment:
To get more useful error, we can use `assert.match`
```suggestion
  match(result.stderr, /Error \[ERR_MODULE_NOT_FOUND\]:/);
```